.golangci.yml

```diff
@@ -138,9 +138,8 @@ linters:
     - errname
     - errorlint
     - errcheck
-    - exportloopref
+    - copyloopvar
```

Comment:
Question: is exportloopref no longer supported?

Reply:
I did this change due to the following warning: level=warning msg="The linter 'exportloopref' is deprecated (since v1.60.2) due to: Since Go1.22 (loopvar) this linter is no longer relevant. Replaced by copyloopvar."